crates/proof-of-sql/src/sql/proof_exprs/numerical_util.rs

```diff
@@ -132,6 +134,16 @@ pub(crate) fn multiply_columns<'a, S: Scalar>(
     })
 }
 
+/// Convert column to scalar slice.
+#[cfg_attr(not(test), expect(dead_code))]
+#[allow(clippy::missing_panics_doc)]
+pub(crate) fn columns_to_scalar_slice<'a, S: Scalar>(
```

Comment:
I generally don't like this type of function that deletes the type information of a column.
Since I'm guessing you are using it in a follow-up PR, just include it with that one, so that I can see if we really need it.